stixcore/data/test.py

```diff
@@ -0,0 +1,31 @@
+import tarfile
+from pathlib import Path
+from urllib.request import urlopen
+
+data_dir = Path(__file__).parent
+test_dir = data_dir.joinpath('test')
+
+__all__ = ['data_dir', 'test_dir']
+
+TEST_DATA_BASE_URL = 'https://homepages.dias.ie/smaloney/stix-data/'
+TEST_DATA_NAME = 'test_data.tar.gz'
+
+
+def _download_test_tar(overwrite=False):
+    destination = data_dir / TEST_DATA_NAME
+    if destination.exists() and overwrite is False:
+        return
+    with urlopen(TEST_DATA_BASE_URL+TEST_DATA_NAME) as req:
+        with destination.open(mode='wb') as file:
+            file.write(req.read())
```

Comment:
maybe use:

urllib.request.urlretrieve

Reply:
`urllib.request.urlretrieve` considered legacy maybe removed in the future https://docs.python.org/3.6/library/urllib.request.html#legacy-interface